Scripts/Mac/BuildInstaller.sh

```diff
@@ -0,0 +1,89 @@
+#!/bin/bash
+
+. "$(dirname ${BASH_SOURCE[0]})/InitializeEnvironment.sh"
+
+CONFIGURATION=$1
+if [ -z $CONFIGURATION ]; then
+  CONFIGURATION=Debug
+fi
+
+PACKAGEVERSION=$2
+if [ -z $PACKAGEVERSION ]; then
+  PACKAGEVERSION="0.0.0.1"
+fi
+
+INSTALLROOT=$VFS_OUTPUTDIR/"InstallerRoot"
+VFSFORGITDESTINATION="usr/local/vfsforgit"
+KEXTDESTINATION="$VFSFORGITDESTINATION"
```

Comment:
I don't think the kext files should go in the same folder as the rest of the VFS for Git files. At the very least, I would create a subdirectory for them.

Reply:
Though again, look at the pattern on Windows. We lay down the driver and its lib in a subfolder, though we will also need to copy the lib into a location that allows the GVFS.Mount process to find it.

Reply:
Moved the Kext to /Library/Extensions. Testing /usr/local/lib directory for the libs.